sdk/typespec/typespec_client_core/src/http/options/mod.rs

```diff
@@ -27,6 +42,8 @@ pub struct ClientOptions {
 
     /// Transport options.
     pub transport: Option<TransportOptions>,
+
+    pub logging: LoggingOptions,
```

Comment:
Comment.

But, yes, I agree with this. And makes me wonder if we should also change `TransportOptions` from an `Option`. It could instead default to a `Default` variant or something. These all take up the same amount of space either way. Treating some sub-client options as structs and others as `Option` seems weird.

Don't change it here - your code is fine - but I wanted to add this and get thoughts. /cc @RickWinter @ronniegeraghty @chlowell

Reply:
I opened #2956 to track.